File: cloud/dynamic-site/gt-cs-1331-py/app/solutions/solution104.py
from solution_api.solution import Solution

class Solution104(Solution):
    id = "1.4"
    title = "Table"
    is_pattern = True
    table = ["\t".join(["a", "a^2", "a^3"])]
    table.extend(list(map(lambda i: "\t".join([str(i),str(i*i),str(i**3)]), range(1,5))))
    answer = table

File: cloud/dynamic-site/gt-cs-1331-csharp/solutions/Solution104.cs
using _4MeOnWeb.SolutionAPI;

namespace _4MeOnWeb.Solutions {
  public class Solution104 : ImmutableSolution {
    public Solution104() :
        base("1.4", "Table") {
    }

    public override IEnumerable<string> Answers =>
      new string[] { "a\ta^2\ta^3" }.Concat(
        Enumerable.Range(1, 4).Select(x => String.Join("\t",
          new string[] { $"{x}", $"{Math.Pow(x, 2)}", $"{Math.Pow(x, 3)}" })
        ));
  }
}